tests/index.html

```diff
@@ -12,6 +12,7 @@
 <meta charset="utf-8">
 <script>
   (window.customElements = window.customElements || {}).forcePolyfill = true;
+  window.customElements.enableCustomizedBuiltins = true;
```

Comment:
https://github.com/webcomponents/custom-elements/pull/88/files#diff-2a35783ddd795619bc384b6c770fe627R90 is a test that does just that